### PR TITLE
update header display logic

### PR DESCRIPTION
**Pull request by averybross**

### CODE REVIEW

**Comment by Ely-S** [comment]
Can you also remove this button? it's now redundant 
image

**Comment by Ely-S** [comment]
If you could also add the new logo in this PR it would be great

**Comment by averybross** [comment]
> If you could also add the new logo in this PR it would be great

added!

**Comment by averybross** [comment]
@Ely-S 

image

image


This is what the root route and /app route would look like then you're not logged in

File: app/components/Header.tsx [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
-          })}
-        >
-          <Link to="/app">Enter App</Link>
-        </button>
-        <div
-          className={clsx("", {
-            hidden: location.pathname === "/",
-          })}
-        >
+        <div style={{ display: "inline-block" }} className="mr-3">
```
Comment (ryanlabouve) [inline app/components/Header.tsx:23]: `.inline-block` if you want to use tailwind

File: app/components/Header.tsx [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
           <ConnectWallet />
         </div>
+        <div style={{ display: "inline-block" }}>
```
Comment (ryanlabouve) [inline app/components/Header.tsx:26]: `.inline-block` if you want to use tailwind

File: app/components/Header.tsx [hunk 3]
```diff
@@ -1,29 +1,35 @@
...
+          {location.pathname === "/" && chainId ? (
```
Comment (ryanlabouve) [inline app/components/Header.tsx:27]: redundant ternary. since you are doing null you can consider  

```
{location.pathname === "/" && chainId && <>
```

**Review by Ely-S — APPROVED** [review]

**Comment by Ely-S** [comment]
Thanks Avery!

**Pull request merged**